index.html

```diff
@@ -29,68 +29,56 @@ <h3><a href="#">MENU</a></h3>
 	
 	
 	
-	
-	
 	<!-- HEADER -- HERO IMAGE AND INTRO SECTION -->		
 	<header>
 		<section class="sectionHeader">
-			<h1 id="headerUpper">Hello</h1>
+			<h1 id="headerUpper">Hello,</h1>
 			<h1 id="headerLower">I'm Ayannah</h1>
 			<h2>I am a full-stack developer</h2>
 		</section>
-		
 	</header>
 	<!-- /HEADER -->
 	
-	
-	
-	
-	
-	
-	
-	
-	
-	
-	
+
 	
 	<!-- MAIN -- ABOUT SECTION -->	
 	<main id="aboutMe">
 		<section class="mainSection">
-			<div>
-				<p>about me</p>
+			<div calss="mainInfo">
+				<h2>ABOUT ME</h2>
+				<p>I am currently enrolled on the Full-Stack Developer course at iO Academy and will be graduating in April 2023.</p>
+				<p>So far, I have learned how to use HTML and CSS for front-end development.</p>
+				<p>Prior to attending the course, I went to University to study Information Technology Management and Business. I also worked in an IT company for 3 years, where my role was a Junior Product Consultant.</p>
+			</div>
+			<div class="mainImage">
+			<img src="images/avatar.png" alt="avatar">
 			</div>
 		</section>
 	</main>
 	<!-- /HEADER -->
 	
 	
 	
-	
-	
-	
-	
-	
-	
-	
-	
 	<!-- ARTICLE -- PORTFOLIO WORK SECTION-->
 	<article id="portfolio">
 		<section class="articleHeader">
 			<h2 class="articleLogo">MY PORTFOLIO</h2>
 		</section>
 		<section class="articleSection">
```

Comment:
For the articleSection on the mobile version, it may be a better solution to change the flex-direction to column (?)